scripts/build/arch/parisc.sh

```diff
@@ -0,0 +1,14 @@
+# Compute parisc-specific values
+
+CT_DoArchTupleValues()
+{
+    # The architecture part of the tuple:
+    CT_TARGET_ARCH="${CT_ARCH}${CT_ARCH_SUFFIX}"
+}
+
+CT_DoArchUClibcConfig()
```

Comment:
Not sure why this is here. uclibc is not currently supported for this architecture.